kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DomainMetadataSuite.scala

```diff
@@ -540,4 +543,97 @@ class DomainMetadataSuite extends DeltaTableWriteSuiteBase with ParquetSuiteBase
       }
     }
   }
+
+  test(
+    "RowTrackingMetadataDomain is serializable and deserializable"
+  ) {
+    withTempDirAndEngine((tablePath, engine) => {
+      // Create a RowTrackingMetadataDomain
```

Comment:
nit: IMO, you can avoid superfluous comments like this. The next line is self-descriptive : it creates a new object

Reply:
Yeah I agree, thanks for pointing this out! I’ve removed similar comments and made the code more self-descriptive.